src/Compile.hs

```diff
@@ -464,3 +482,24 @@ curryPowder = unlines
   , "};"
   , ""
   ]
+
+eq :: String
+eq = unlines
```

Comment:
Find a way to move these out of each built module, build them once in the destination folder and resolve imports to it.

Reply:
Ticket linked above for that